composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": ">=5.3.0",
         "illuminate/support": "5.0.* || 5.1.* || 5.2.* || 5.3.* || 5.4.* || 5.5.*",
-        "omnipay/common": "~2.0"
```

Comment:
This doesn't belong in this pr

Reply:
@barryvdh Thank you for pointing this out. Reverted back to the original.